.tekton/pull-request.yaml

```diff
@@ -4,7 +4,7 @@ kind: PipelineRun
 metadata:
   name: build-definitions-pull-request
   annotations:
-    pipelinesascode.tekton.dev/on-cel-expression: (event == "pull_request" && target_branch == "main") || (event == "push" && target_branch.startsWith("gh-readonly-queue/main/"))
+    pipelinesascode.tekton.dev/on-cel-expression: (event == "pull_request" && target_branch == "main" && ( !has(body.pull_request) || body.pull_request.draft == false) ) || (event == "push" && target_branch.startsWith("gh-readonly-queue/main/"))
```

Comment:
`!has(body.pull_request)` is required, otherwise it fails on
```
there was an error evaluating the CEL expression, skipping: expression \"(event == \\\"pull_request\\\" && target_branch == \\\"main\\\" && body.pull_request.draft == false) || (event == \\\"push\\\" && target_branch.startsWith(\\\"gh-readonly-queue/main/\\\"))\" failed to evaluate: no such key: pull_request"
```
when commenting `/(re)?test`. I assume this is happening since such event is not a type of `pull_request`, but `issue_comment` ([docs](https://docs.github.com/en/webhooks/webhook-events-and-payloads#issue_comment))